static/scss/components/banners.scss

```diff
@@ -333,3 +333,49 @@
     }
 
 }
+
+
+.c-footer-banner {
+    background-color: $color-white;
+
+    .mzp-l-content {
```

Comment:
Isn't `.mzp-*` indicative of Protocol classes? If so, seems unwise to be modifying those - it might clash with future changes there, or with developer expectations for other users of those classes. (If not, ignore this :) )

Reply:
Oof. I was feeling skirmy about this when I dropped it in. I was wanting to use Protocol, but had to override basically all the padding. I dropped the div/classes entirely and just used tokens to get the max-width to match.